[yoga] Adds features that are helpful when deploying Trilio Vault

jovial opened the pull request.

markgoddard commented on ansible/roles/kolla-ansible/defaults/main.yml:
```diff
@@ -210,6 +210,8 @@ kolla_openstack_logging_debug:
 # Nova options
 #######################
 
+kolla_nova_compute_ironic_host:
```
Needs a comment similar to the one below, and the commented out version in etc/kayobe/nova.yml.

markgoddard commented on ansible/roles/kolla-ansible/templates/overcloud-services.j2:
```diff
@@ -37,11 +37,16 @@ nova
+{{ kolla_nova_compute_ironic_host }}
+{% else %}
+[nova-compute-ironic:children]
 {{ kolla_nova_compute_ironic_group }}
```
The default group doesn't need to be configurable - can we set this to just `nova`, then remove all references to kolla_nova_compute_ironic_group?

autonomouse replied:
done

markgoddard committed:
Support --kolla-playbook argument

Adds support for specifying a custom playbook when running Kolla Ansible
commands via a '--kolla-playbook' argument.

Change-Id: Ie4bba0f1bdc82657572e0f06a71db1140c0bd3a4
(cherry picked from commit 79a8610030b214dd013f167b66464b79be937f67)

Alex-Welsh committed:
added option to skip hooks

adds the argument --skip-hooks/-sh which will stop the execution of
hooked ansible playbooks. Either a pattern can be specified to match
against or hook execution can be stopped altogether with "all"

Story: 2009241
Task: 43390

Change-Id: I4f2176aa056fec62e31d07140e3d05779480a93d
(cherry picked from commit d7069283fd89e6164414d83afa688c3d2afae48f)
